scripts/kdb-bash-completion

```diff
@@ -23,7 +23,7 @@ _kdb() {
 	# only kdb was entered yet, print a list of available commands
 	if [[ $COMP_CWORD -eq 1 ]]; then
 		local IFS=$'\n'
-		local commands=($(${kdbpath} 2> /dev/null | sed -e '0,/^Known commands are/d' | awk '{print $1}'))
+		local commands=($(${kdbpath} 2> /dev/null | sed -e '0,/^Known commands are/d' | awk '{print $1}' | sed -r "s/\x1B\[([0-9]{1,3}((;[0-9]{1,3})*)?)?[m|K]//g" ))
```

Comment:
I think you can use the `kdb` subcommand [`list-commands`](https://www.libelektra.org/manpages/kdb-list-commands) to simplify this code: 

```suggestion
		local commands=($(${kdbpath} list-commands))
```

.